introduction.md

```diff
@@ -3,3 +3,8 @@ Hello,
 My name is Eddie. I am working on Mastering GitHub.
 
 Electives probably, maybe CSS.
+
+* Git
+* CSS
+* Sass
+* HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?